requirements.txt

```diff
@@ -0,0 +1,11 @@
+python
+
+numpy
+ipopt
+cython
+setuptools
+pkg-config
+six
+future
```

Comment:
Probably should put some minimum bounds on these.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/1edb9ab9bbee110a3ca3304baebdcbe2500507e0 and https://github.com/matthias-k/cyipopt/pull/75/commits/b27d747f4ca6280f9f34f41d6f31a230a48917c3.